experiments/test2_replacement_strategies.py

```
@@ -0,0 +1,87 @@
+    shutil.copy("test_assets/test_clean.jpg", test_file)
+    
+    # Étape 1: Ajouter métadonnées initiales avec doublons
+    print("\n1. Ajout métadonnées initiales avec doublons...")
+    cmd1 = [
+        "exiftool", "-overwrite_original",
+        "-XMP-iptcExt:PersonInImage=Anthony",
+        "-XMP-iptcExt:PersonInImage+=Bernard", 
+        "-XMP-iptcExt:PersonInImage+=Anthony",  # Doublon volontaire
+        str(test_file)
+    ]
+    subprocess.run(cmd1, capture_output=True)
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)